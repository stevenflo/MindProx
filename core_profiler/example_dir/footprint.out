# Mix output version 2
# FINI: end of program
# EMIT_DYNAMIC_STATS FOR TID 1
# instructions = 99232, basic blocks = 21998
Final 90% static basic block count is 372, final dynamic basic block count is 20082


Data statistics
Total page count is 37005
Total touched static pages, size of pagecount_vector is 75
Final 90% static data page count is 26, final dynamic page count is 33801
Final 50% static data page count is 3


# END_DYNAMIC_STATS
