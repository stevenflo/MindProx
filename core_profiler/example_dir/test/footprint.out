# Mix output version 2
# FINI: end of program
# EMIT_DYNAMIC_STATS FOR TID 1
# instructions = 96442, basic blocks = 21210
Final 90% static basic block count is 344, final dynamic basic block count is 19286


Data statistics
Total page count is 36254
Total touched static pages, size of pagecount_vector is 75
Final 90% static data page count is 25, final dynamic page count is 32916
Final 50% static data page count is 3


# END_DYNAMIC_STATS
